File: Ruby/FizzBuzz.rb
def FizzBuzz(max)
    for i in 1..max
        output = ""
        if i % 3 == 0
            output = "Fizz"
        end

        if i % 5 == 0    
            output += output == "" ? "Buzz" : " Buzz"
        end

        if output == ""
            output = i
        end

        puts output
    end
end

if __FILE__ == $0
    FizzBuzz(100)
end

File: Lua/FizzBuzz.lua
function FizzBuzz(max)
    for i=1, max do
        output = ""
        if(i%3==0) then
            output = "Fizz"
        end

        if(i%5==0) then
            if(output~="") then
                output = output .. " "
            end
            output = output .. "Buzz"
        end

        if(output == "") then
            output = i
        end

        print(output)
    end
end

print(FizzBuzz(100))
